src/CompletionProvider.php

```diff
@@ -310,26 +314,22 @@ public function provideCompletion(PhpDocument $doc, Position $pos): CompletionLi
             // Suggest global symbols that either
             //  - start with the current namespace + prefix, if the Name node is not fully qualified
             //  - start with just the prefix, if the Name node is fully qualified
-            foreach ($this->index->getDefinitions() as $fqn => $def) {
+            foreach ($this->index->getGlobalDefinitions() as $fqn => $def) {
```

Comment:
Because there was previously a loop on all definitions, with a [check to keep only the global ones](https://github.com/felixfbecker/php-language-server/pull/451/files/1fec8082395e7ce47a905e9acd2d2dedfea50527#diff-18eb38922fb7235266f15163e2b050edL319). By using this new function, we can reduce the loop size by having directly the global definitions (and so return a result faster).